module/item/systems/repair.js

```diff
@@ -274,25 +339,29 @@ class RepairData {
     }
 
     get repairDC() {
-        return this.diagram.system.craftingDC - repairModifier + this.enchantsDC
+        return this.diagram.system.craftingDC + repairModifier + this.enchantsDC
     }
 
-    get repairDCForumulae() {
-        let formulae = `${this.diagram.system.craftingDC}[${game.i18n.localize('WITCHER.Repair.params.craftingDC')}] - ${repairModifier}[${game.i18n.localize('WITCHER.Repair.params.repairMod')}]`
+    get repairDCForumula() {
+        let formula = `${this.diagram.system.craftingDC}[${game.i18n.localize('WITCHER.Repair.params.craftingDC')}] - ${Math.abs(repairModifier)}[${game.i18n.localize('WITCHER.Repair.params.repairMod')}]`
         const echantsCount = this.enchantsCount
```

Comment:
typo

Reply:
Which one? Thought I corrected the typo here - was an unnecessary plural form

Reply:
Forumula, echantsCount